### PR TITLE
Add french translation of the plugin

### PR DESCRIPTION
**Pull request by Vlammar**
Add a first french translation, I wasn't sure on some part so it may need some refining afterhand. Some item description on the royal horn are missing and that make the translation harder because I don't know what those item are used for. If you need more help to translate the plugin from english to french I'm still available. (I'm a french native speaker)
e.g radio case can be translated in to things in french => a box that contains a radio or a type of radio that is old schooled and squared

Linked to issue #81 and to PR #82

### CODE REVIEW

File: assets/rpvoicechat/lang/fr.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    "Command.Talk.Desc": "Défini la distance de parole en blocs",
+    "Command.Talk.Success": "Distance de parole défini à {0}",
+    "Command.Whisper.Desc": "Défini la distance de chuchotage en blocs",
+    "Command.Whisper.Success": "Distance de chuchotage défini à {0}",
+    "Command.Info.Desc": "Affiche les distances audio actuelles",
+    "Command.Info.Success": "Distance de chuchotage: {0} blocs\nDistance de parole: {1} blocs\nDistance de cri: {2} blocs",
+    "Command.Reset.Desc": "Réinitialise les distances audio",
+    "Command.Reset.Success": "Distances audio réinitialisées",
+    "Hotkey.ModMenu": "RPVoice: Menu du mod",
+    "Hotkey.VoiceLevel": "RPVoice: Changer la distance de la voix voice distance",
```
Comment (Dmitry221060) [inline assets/rpvoicechat/lang/fr.json:27]: Shouldn't it be `RPVoice: Changer la distance de la voix`?

File: assets/rpvoicechat/lang/fr.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    "Gui.ModMenu.outputGain.Label": "Volume des joueurs",
+    "Gui.ModMenu.outputGain.Tooltip": "Augmentation du volume des autres joueurs",
+    "Gui.ModMenu.inputGain.Label": "Volume d'enregistrement",
+    "Gui.ModMenu.inputGain.Tooltip": "Changer votre volume",
+    "Gui.ModMenu.inputThreshold.Label": "Seuil de sensibilité de l'entrée audio",
+    "Gui.ModMenu.inputThreshold.Tooltip": "A quel seuil votre audio commence à transmettre",
+    "Gui.ModMenu.audioMeter.Label": "Audiomètre",
+    "Gui.ModMenu.audioMeter.Tooltip": "Affiche votre amplitude audio",
+    "Gui.ModMenu.toggleHUD.Label": "Active/Désactive l'ATH",
+    "Gui.ModMenu.toggleHUD.Tooltip": "Active/désactive la visibilité  la visibilité des éléments de l'ATH ",
```
Comment (Dmitry221060) [inline assets/rpvoicechat/lang/fr.json:53]: Is `l'ATH` a common term in french to be recognized as `HUD`?

File: assets/rpvoicechat/lang/fr.json (hunk 1 above)
Comment (Vlammar) [reply assets/rpvoicechat/lang/fr.json:27]: yeah I totaly forgot to change this

**Review by Dmitry221060 — APPROVED** [review]
Tested it in game, looks great 👍
 
I've changed base branch to net7 to not merge one feature branch into another, so make sure to merge #82 first.

**Pull request merged**